src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -190,6 +199,7 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         private courseAccessStorageService: CourseAccessStorageService,
         private profileService: ProfileService,
         private modalService: NgbModal,
+        private renderer: Renderer2,
```

Comment:
This seems unused to me, I think we can remove that attribute and the import

```suggestion
```

Reply:
agreed